lib/features/mailbox_creator/presentation/extensions/validator_failure_extension.dart

```diff
@@ -50,4 +50,14 @@ extension ValicatorFailureExtension on VerifyNameFailure {
       return '';
     }
   }
+
+  String getMessageForwarding(BuildContext context) {
+    if (exception is EmptyNameException) {
+      return AppLocalizations.of(context).this_field_cannot_be_blank;
```

Comment:
Why we don't name it lowerCamelCase

Reply:
It's an existence of the first version, it's involved in many places so we don't want to fix it now